change/@griffel-react-6d0f6f43-4864-4ba4-b292-8e46b9d1d20f.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

```suggestion
  "type": "none",
```
beachball will rollup the changelog from core